packages/Utils/hex-to-rgba.ts

```diff
@@ -1,8 +1,14 @@
 import { hexToRGB } from './hex-to-rgb'
 
+const colorFormatRegex =
+  /^(rgb|hsl)(a?)[(]\s*([\d.]+\s*%?)\s*,\s*([\d.]+\s*%?)\s*,\s*([\d.]+\s*%?)\s*(?:,\s*([\d.]+)\s*)?[)]$/
+
 export const hexToRGBA = (hex: string, transparency = 1): string => {
   if (!hex) return
 
+  // check if it's rgb(), rgba(), hsl() and hsla()
+  if (!hex.startsWith('#') && colorFormatRegex.test(hex)) return hex
```

Comment:
I think we should just check if it doesn't starts with `#` otherwise it won't be futur proof, see: https://web.dev/state-of-css-2022/#color-level-4-and-5